src/utils/time.js

```diff
@@ -46,6 +46,21 @@ export const daysAfter = (date, days) => {
   return new Date(date.getTime() + days * DAY)
 }
 
+export const daysBeforeDateRangeExcWeekend = (date, days) => {
+  let daysCountdown = Array(7).fill('')
+  daysCountdown.forEach((_, index) => {
+    daysCountdown[index] = new Date(date.getTime() - days * DAY)
+    days -= 1
+  })
+
+  const removeWeekends = daysCountdown.filter(
+    (date) =>
+      date.toString().slice(0, 3) !== 'Sat' &&
+      date.toString().slice(0, 3) !== 'Sun'
+  )
+  return removeWeekends.reverse()
+}
```

Comment:
The name is a little confusing. Maybe something more like `getWorkingDaysBeforeDate`. And maybe rename days to `numberOfDays`.